src/common/options.cc

```diff
@@ -7220,6 +7220,40 @@ std::vector<Option> get_rgw_options() {
     .add_see_also("rgw_dynamic_resharding")
     .add_see_also("rgw_max_objs_per_shard"),
 
+    Option("rgw_dynamic_resharding_static_shards", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(0)
+    .set_description("The target number of shards for any dynamically resharded bucket")
+    .set_long_description(
+        "Dynamic resharding takes linear time to reshard buckets relative the "
+        "number of index entries in that bucket's index. For clusters with an "
+        "exponentially distributed number of entries per bucket and a small "
```

Comment:
I'm not quite clear on what you mean by "exponentially" here. If the x-axis is the number of objects in a bucket, and the y-axis is the number of buckets with that size, it would seem to be more of a negative power function.

Reply:
I was thinking of an exponential distribution's probability density function. Either concept applies pretty well, but I can nix the talk of distributions if that's distracting from the point.